src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirectiveTargetExtension.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
     public class InjectDirectiveTargetExtension : IInjectDirectiveTargetExtension
     {
-        private const string RazorInjectAttribute = "[global::Microsoft.AspNetCore.Razor.Internal.RazorInjectAttribute]";
+        private const string RazorInjectAttribute = "[global::Microsoft.AspNetCore.Mvc.Razor.Internal.RazorInjectAttribute]";
```

Comment:
When you have this kind of change in the future do everyone a favour and send it out as a separate PR. It's very easy to review/approve mechanical changes like this.